internal/runtime/runner.go

```diff
@@ -71,7 +72,7 @@ func NewRunner(testers []Tester, settings e2e.Settings) *Runner {
 func (r *Runner) Run() error {
 	for _, scenario := range r.spec.Scenarios {
 		scenarioTag := r.generateScenarioTag()
-		r.logger.Debugf("[scenario]: %s, [Tag]: %s", scenario.Description, scenarioTag)
+		r.logger.Debugf("[scenario]: %s, [Tag]: %s", strings.TrimSpace(scenario.Description), scenarioTag)
```

Comment:
what is the reason for this change?

Reply:
There is a hanging newline in `scenario.Description` that splits this line into two lines unintentionally. This is purely a readability change on the debug logs.